_dashboards/maps.md

```diff
@@ -0,0 +1,148 @@
+---
+layout: default
+title: Maps
+nav_order: 15
+---
+
+# Maps
+
+With OpenSearch Dashboards, you can create maps to visualize your geographical data. OpenSearch lets you construct map visualizations with multiple layers, combining data across different indexes. You can build each layer from a different data source either directly or using simple or ML-powered aggregations. Additionally, you can configure maps to show specific data at different zoom levels. OpenSearch maps are powered by the OpenSearch maps service, which uses vector tiles to render maps. 
+
+## Getting started
+
+To get started, perform the following steps:
+
+1. On the top menu bar, go to **OpenSearch Dashboards > Maps**.
+1. Select the **Create map** button. You can now see the default OpenSearch basemap.
+1. To examine the **Default map** layer configuration, in the **Layers** panel on the upper left of the map, select **Default map**.
+![Default map]({{site.url}}{{site.baseurl}}/images/maps/maps-default.png)
+
+To hide the **Layers** panel, select the collapse (arrow) icon in the panel's upper-right corner.
+{: .tip}
+
+## Layer settings
+
+To change the default map settings, select **Default map** in the **Layers** panel. Under **Layer settings**, you can change the layer name and description and configure zoom levels and opacity for your layer:
+
+- **Zoom levels**: If you want to make a layer visible only on a certain range of zoom levels, you can specify the zoom levels for this layer either by entering them in the text boxes or by sliding the range slider to the desired values.
```

Comment:
Shall we call out like, by default layer is visible in all zoom levels, you can update this range if you want otherwise.